eel/src/keys_manager.rs

```diff
@@ -37,12 +32,12 @@ pub fn mnemonic_to_secret(mnemonic_string: Vec<String>, passphrase: String) -> R
 }
 
 fn derive_secret_from_mnemonic(mnemonic: Mnemonic, passphrase: String) -> Secret {
```

Comment:
I think it makes sense to move this logic (secret generation, mnemonic to secret, etc) from `eel` to `3L`. But let's do that in another PR.

Reply:
I've created a Jira task.